src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
don't reset margins and paddings using * selector
reset it only for needed elements